### ISSUE TITLE
numbro().unformat() is not a function, but numbro.unformat() works

### ISSUE THREAD

**Issue by josh803316** [BenjaminVanRyseghem/numbro#339]
Per the [docs](http://numbrojs.com/format.html#unformat), we should be using numbro().unformat() but when I try that with the latest current version (after upgrading -- see diff below)

`-    "numbro": "^1.11.1",
+    "numbro": "^2.0.5",` 

I'm running into errors like this:
TypeError: (0 , _numbro.default)(...).unformat is not a function.  

I'm using numbro like this:
```javascript
import numbro from 'numbro'; 
propertyPreprocessor: (enteredProperty) =>  numbro().unformat(numbro(enteredProperty).format('3a'))

 \\ This works:
propertyPreprocessor: (enteredProperty) =>  numbro.unformat(numbro(enteredProperty).format('3a'))```

**Comment by BenjaminVanRyseghem** [comment]
indeed, this behavior was changed in 2.x and unfortunately I hadn't had the time yet to update the doc.

**Comment by josh803316** [comment]
Thanks for the heads up, didn't want to code a bunch of changes that would be reverted again later... much appreciated!
